tests/features/eap/7/eap_7_resourceadapters.feature

```diff
@@ -30,15 +30,15 @@ Feature: EAP 7 Openshift resource adapters
     Then XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value activemq-rar.rar on XPath //*[local-name()='resource-adapter']/@id
      And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value activemq-rar.rar on XPath //*[local-name()='resource-adapter']/*[local-name()='archive']
      And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value org.apache.activemq.ra.ActiveMQManagedConnectionFactory on XPath //*[local-name()='resource-adapter']/*[local-name()='connection-definitions']/*[local-name()='connection-definition']/@class-name
-     And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value java:/ConnectionFactory on XPath //*[local-name()='resource-adapter']/*[local-name()='connection-definitions']/*[local-name()='connection-definition']/@jndi-name
+     And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value java:/ConnectionFactory on XPath //*[local-name()='resource-adapter']/*[local-name()='connection-definitions']/*[local-name()='connection-definition']/jndi-name
```

Comment:
sure we want to do this?

Reply:
Good, catch, thanks.

Reply:
Thx, fixed. I checked the other instances of this too and they're fine, so odd this one got caught up. It was probably an accident from initially starting these substitutions with perl -i -e 's/..../.../g' then realising all those @'s in perl were just going to be a pain and switching to sed ;)